ftpvl/fetchers.py

```diff
@@ -114,17 +114,47 @@ def _download(self) -> List[Dict]:
             raise IndexError(f"Invalid eval_num: {self.eval_num}")
         build_nums = evals_json["evals"][self.eval_num]["builds"]
 
-        # collect the 'meta.json' build products
+        # fetch build info and download 'meta.json'
         data = []
         for build_num in build_nums:
+            # get build info
             resp = requests.get(
-                f"https://hydra.vtr.tools/build/{build_num}/download/1/meta.json",
+                f"https://hydra.vtr.tools/build/{build_num}",
+                headers={"Content-Type": "application/json"},
+            )
+            if resp.status_code != 200:
+                raise Exception(f"Unable to get build {build_num} due to non-200 status code.")
+            
+            decoded = None
+            try:
+                decoded = resp.json()
+            except json.decoder.JSONDecodeError:
+                raise Exception(f"Unable to decode build {build_num} JSON file.")
```

Comment:
Again, may be useful to surface some human-readable part of the JSONDecodeError in the error message, for easier debugging purposes.